mesop/components/bokeh_plot/bokeh_plot.py

```diff
@@ -0,0 +1,29 @@
+from io import BytesIO
+from typing import Protocol
+from bokeh.plotting import figure
+from mesop.component_helpers import Style, component
+from mesop.components.html.html import html
+from bokeh.embed import file_html
```

Comment:
Rather than importing here, which will make everyone using Mesop depend on bokeh, could you import bokeh  inside the bokeh_plot function? (i.e. L29).